ground/src/test/java/com/google/android/ground/repository/MapStateRepositoryTest.kt

```diff
@@ -30,6 +33,7 @@ import org.robolectric.RobolectricTestRunner
 class MapStateRepositoryTest : BaseHiltTest() {
 
   @Inject lateinit var mapStateRepository: MapStateRepository
+  @Inject lateinit var localValueStore: LocalValueStore
 
   @Test
   fun getMapType_whenNotSet_returnsDefault() {
```

Comment:
Can convert these to string function names and below too! Thank you!

Reply:
done